### DISCUSSION TITLE
QExpansionItem: some items non-expandable, but not disabled

### DISCUSSION THREAD

**Discussion by andrisi** [quasarframework/quasar#14813] [General - Components / Directives / etc]
I have a list, and some items are expandable, but others are not, and this might even change after the items are displayed. Is there a way to set a QExpansionItem so that it does not have an expand item, and some others with it? I would not set it to disabled as the item itself is something the user can interact with, so the "disabled" hover over mouse pointer would not be suitable.

**Comment by MilosPaunovic** [comment]
You could use header slot to achive what you need.
https://quasar.dev/vue-components/expansion-item#example--header-slot
